mitodl/odl-video-service
# Lecture Capture: move unrecognized videos into an admin-only collection

#### What are the relevant tickets?
Closes #690

#### What's this PR do?
Sends videos in the watch bucket with bad filenames to a default 'Unsorted' collection.

#### How should this be manually tested?
- Manually upload a video with a bad filename to the S3 watch bucket:
  ```python
  from django.conf import settings
  import boto3
  s3 = boto3.resource('s3')
  s3.meta.client.upload_file('<your_video_file.mp4>', settings.VIDEO_S3_WATCH_BUCKET, 'A bad filename.mp4')
  ```
- Wait for the `monitor_watch_bucket` task to run (or run it manually).
- After it runs, the watch bucket should be empty and there should be a new `Unsorted` collection containing the video `A bad filename.mp4`
